### ISSUE TITLE
How Can I Remove A Visual?

### ISSUE THREAD

**Issue by Emrullah-Akbulak** [datoviz/datoviz#68]
I have two panels and I have added few visuals to these panels. I want to remove some of these visuals from these panels. How can I do that?

Also I am having a problem when I update a visual too frequently. I am calling the dvz.visual_update function to update the visual 60 times a second to render some data I have and sometimes the program just seg. faults and exits. 

I assumed that since the visual_update function is non-blocking I might be trying to update the visual before the previous update request finishes but I am not sure. Unfortunatelly I cannot share my code but it would be awesome if we can have an example on how to remove a visual and how to frequently update a visual correctly.

I would like to thank you for the library as well 🙂

**Comment by rossant** [comment]
- For now, best is to use `dvz.visual_show(visual, False)`, see the `panels.py` example.
- You should not need to call `visual_update()`: updates should occur automatically when you update your visual. I added an [example here](https://github.com/datoviz/datoviz/blob/main/examples/mesh.py#L72).

Please let me know if you still have issues!

**Comment by Emrullah-Akbulak** [comment]
Thank you for the quick response and help! Removing the visual_update statements seems to solve the problem. It was a real headache to debug.

I guess I misunderstood the C Api documentation. I saw the update statement in "examples/gui.py" example  and stick with it in the rest of my code and since it did not really effected my code visibly at first and the c api explanation literally said "update a visual after its data has changed" I did not think about checking it.
